specification/mediaservices/resource-manager/Microsoft.Media/stable/2021-06-01/Encoding.json

```diff
@@ -1351,6 +1351,10 @@
         }
       ],
       "properties": {
+        "configurations": {
+          "$ref": "#/definitions/PresetConfigurations",
+          "description": "Optional configuration settings for encoder. Configurations is only supported for ContentAwareEncoding and H265ContentAwareEncoding BuiltInStandardEncoderPreset."
```

Comment:
Should say "PresetConfigurations are only supported for ContentAwareEncoding, H265ContentAwareEncoding, and the BuiltInStandardEncoderPreset.  These settings will not affect any other presets, or custom presets

Reply:
updated with PresetConfigurations are only supported for the ContentAwareEncoding and H265ContentAwareEncoding built-in presets. These settings will not affect other built-in or custom defined presets.